README.md

```diff
@@ -137,7 +138,7 @@ git commit -m 'change the url name.'
 
 
 
-#### Publish to Your gh-pages Branch
+#### Publish Your gh-pages Branch to Your cesehub Folk on GitHub)
 
```

Comment:
@yuhow did you really mean "Folk"?  Could you explain why do you want to mention any folk on github?

I think you mean parenthesis ")" rather than bracket "]", right?  The right parenthesis looks strange to me.  It doesn't have the corresponding left parenthesis.  Did you forget to add the left one, or the right one is misplaced?